consensus/hotstuff_mempool_test.go

```diff
@@ -0,0 +1,230 @@
+package consensus
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/shared/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+// hotstuffMessageFactory returns a hotstuff message with a block with a single transaction of the given size
+//
+// it is used in tests to make sure that the mempool is correctly limiting the size of the transactions it stores
+//
+// Important: the size of the transaction is not the size of the whole message on the wire, there's obviously some overhead in the protobuf
+func hotstuffMessageFactory(size int) *typesCons.HotstuffMessage {
+	tx := make([]byte, size)
+	for i := range tx {
```

Comment:
Could we do `rand.Read(tx)` here insetad?

Reply:
Good shout, we could but then we couldn't use the same factory method to make assertions on the expected messages in the tests below.

I thought that the simplest, most idiomatic Go way to deterministically create a message of len(n) is to just fill it with a well known sequence. 

I went for: **"All 1s"**

Reply:
Sounds > 💯